plugins/modules/katello_repository_set.py

```diff
@@ -131,13 +140,11 @@
 RETURN = ''' # '''
 
 from ansible.module_utils.foreman_helper import KatelloEntityAnsibleModule
-from ansible.module_utils._text import to_text
 
 
 def get_desired_repos(desired_substitutions, available_repos):
     desired_repos = []
     for sub in desired_substitutions:
-        sub = {to_text(k): to_text(v) for (k, v) in sub.items()}
```

Comment:
Is this replaced by specifying the suboptions, or was it unneeded anyway? (Sensing some unicode related weirdness here)

Reply:
It's replaced by the suboptions, yeah.